digitalpy/core/health/impl/default_health_check.py

```diff
@@ -1,8 +1,8 @@
-from digitalpy.config.configuration import Configuration
-from digitalpy.routing.action_mapper import ActionMapper
-from digitalpy.routing.controller import Controller
-from digitalpy.routing.request import Request
-from digitalpy.routing.response import Response
+from digitalpy.core.digipy_configuration.configuration import Configuration
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module